corehq/warehouse/management/commands/commit_table.py

```diff
@@ -16,7 +16,7 @@
 
 class Command(BaseCommand):
     """
-    Example: ./manage.py stage_table group_staging 222617b9-8cf0-40a2-8462-7f872e1f1344
+    Example: ./manage.py stage_table group_staging 22
```

Comment:
does that id need to be something like a UUID? i would be in favour of keeping the long example so that there is that hint of what it should be.

Reply:
actually this used to be a UUID but is now an integer

Reply:
Perfect.